docs/crio.8.md

```diff
@@ -449,4 +452,4 @@ Shows a list of commands or help for one command
 # SEE ALSO
 
 crio.conf(5), crio.conf.d(5), oci-hooks(5), policy.json(5), registries.conf(5),
-storage.conf(5)
\ No newline at end of file
+storage.conf(5)
```

Comment:
Can you please revert that change? This may fix the docs validation job.

Reply:
There are new options to the criocli for the runtime-storage, so I think they are necessary :/

Reply:
I mean only the newline at the end of the file.

Reply:
Not quite sure where it came from :/